File: 2480.py
s = input().split(" ")
for i in range(3):
	s[i] = int(s[i])


if (s.count(s[0]) == 1 and s.count(s[1]) == 1 and s.count(s[2]) == 1):
	print(max(s) * 100)
elif (s.count(s[0]) == 3):
	print(s[0] * 1000  + 10000)
else:
	if (s.count(s[0]) == 2):
		print(1000 + s[0]*100)
	elif (s.count(s[1]) == 2):
		print(1000 + s[1]*100)
	else:
		print(1000 + s[3]*100)

File: 2480.cpp
#include <stdio.h>
int max(int a, int b, int c){
	if (a > b && a > c){
		return a;
	}
	else if (b > a && b > c){
		return b;
	}
	else if (c > a && c > b){
		return c;
	}
	else{
		return a;
	}
}
int main(){
	int a, b, c;
	scanf("%d %d %d",&a, &b, &c);

	if (a == b && a == c && b == c){
		printf("%d\n", a*1000+10000);
	}
	else if(a != b && a != c && b != c){
		printf("%d\n", max(a, b, c)*100);
	}
	else{
		if((a == b && a != c) || (a == c && a != b)){
			printf("%d\n", 1000 + 100*a);
		}
		else if ((b == a && b != c) || (b == c && b != a)){
			printf("%d\n", 1000 + 100*b);
		}
		else{
			printf("%d\n", 1000 + 100*c);
		}

	}

	return 0;
}
